docs/pages/build/orbit.mdx

```diff
@@ -0,0 +1,34 @@
+---
+title: Expo Orbit
+maxHeadingDepth: 4
+description: Accelerate your development workflow with one-click build launches and simulator management.
+---
+
+import { Terminal } from '~/ui/components/Snippet';
+import Video from '~/components/plugins/Video';
+
+Expo Orbit for macOS makes it faster and easier to install and run builds from EAS or elsewhere, and to run Snack projects on simulators and physical devices.
+
+<Video file="orbit/basic-features.mp4" />
+
+The process for installing builds from EAS (Android device/emulator, iOS simulator/device) and running Snack projects on simulators used to be more manual than it needs to be. You could use `eas build:run` and select a build for Android devices/emulators or iOS simulators, or you could download the archive, extract it, and drag and drop it to the simulator/emulator. For Snack projects, additional steps include installing Expo Go in the simulator through the CLI, logging in, and selecting the Snack from a list. Orbit makes these steps as seamless as possible.
```

Comment:
```suggestion
Before Orbit, installing builds from EAS (Android and iOS physical devices or emulator/simulator) or running Snack projects on simulators was manual. You had to run `eas build:run` command and select a build for your chosen device or download the archive and then drag and drop it to the simulator (in the case of iOS). Also, for Snack projects, additional steps included installing Expo Go on the virtual device, logging in, and then selecting the Snack from the list. Orbit makes all of these steps as seamless as possible. 
```

If we are providing the historical context, and use past tense, let's make sure to use the past tense through the paragraph. I also tried to make it a bit more concise, repeating emulator/simulator too many times was cumbersome to read.